### PR TITLE
488/faq fixes

### LINKED ISSUE #488: FAQ - Typos, comments and enhancements
I will list here typos, inconsistencies etc for the FAQ section

1. CoW vs COW - in the About section all letters are capitalized in the FAQ not should be consistent

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Closes #488 

screenshot_2021-04-26_15-45-15

Skipped 1, 2, 10 and 11 from the list.

No action on 1, 2 and 11.

Probably need a separated issue for 10 with styles and etc

# Testing

Verify items on issue #488 against the `/#/faq` page

### CODE REVIEW

**Review by MareenG — COMMENTED** [review]
All the approved changes have been made. However.,I noticed that often a **.** is missing at the end of the sentences.

**Review by anxolin — APPROVED** [review]
All good, only minor concern is that I'm not sure we should send the user back to the swap page in the **Mik it!** link

This should take us to the post for example.

Otherwise, I suggest just removing the link and leaving it like **Mil it! 🥛**


Merging for now to consolidate. if we need to change sth we can do later.

File: src/custom/pages/Faq/index.tsx [hunk 1]
```diff
@@ -373,13 +384,19 @@ export default function Faq() {
...
 
           <hr />
 
           <p>
-            Didn&#39;t find an answer? Join the <a href={DISCORD_LINK}>community on Discord</a> for support.
+            Didn&#39;t find an answer? Join the <a href={DISCORD_LINK}>community on Discord</a>
+          </p>
+          <p>
+            We really hope you like CowSwap. If you do,&nbsp;<Link to="/">Milk it!</Link>
+            <span role="img" aria-label="happy cow face">
```
Comment (alfetopito) [inline src/custom/pages/Faq/index.tsx:397]: @anxolin 
Update aria-label to match the emoji ;)

Comment (anxolin) [reply src/custom/pages/Faq/index.tsx:397]: 🙈 ahahah

**Pull request merged**